src/bin/sol-fbp-generator/Kconfig

```diff
@@ -1,4 +1,4 @@
 config FBP_GENERATOR
 	bool "fbp generator"
-	depends on SOL_PLATFORM_LINUX && FLOW && RESOLVER_CONFFILE && INSPECTOR
+	depends on SOL_PLATFORM_LINUX && FLOW && NODE_DESCRIPTION && INSPECTOR
```

Comment:
I guess here we need a resolver, not only the descriptions otherwise .conf won't work. Same for runner below.